tests/fixtures/valid.xml

```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE parent SYSTEM "schema.dtd">
```

Comment:
how does this work when the document is loaded into a string? Does it have to indicate an absolute path, a URL?

Reply:
Not checked, but assume it's relative to http://php.net/manual/en/class.domdocument.php#domdocument.props.documenturi.